client/app/bundles/shared/components/Blockquote.scss

```diff
@@ -0,0 +1,24 @@
+@import "_legacy.scss";
+@import "_base.scss";
+
+.text {
+  display: inline-block;
+  position: relative;
+
+  background-color: rgba(255, 255, 255, 0.1);
+  font-family: Lato, sans-serif;
+  font-style: italic;
+  font-size: 36px;
```

Comment:
Let's make the default font size 24px and bump it up to 36px on larger screens -- sorry for missing this the first time!